packages/compiler-cli/src/ngtsc/reflection/src/host.ts

```diff
@@ -395,6 +409,12 @@ export interface BaseDeclaration<T extends ts.Declaration = ts.Declaration> {
    * TypeScript reference to the declaration itself, if one exists.
    */
   node: T|null;
+
+  /**
+   * If set, describes the type of builtin JavaScript declaration this
+   * declaration resolves to.
+   */
+  builtinDecl?: BuiltinDeclaration
```

Comment:
In the ngtsc codebase, we don't use optional structure members (at least not often). Additionally, I think `Decl` as a suffix is a bit redundant given this is in a "Declaration" interface.
```suggestion
  builtin: BuiltinDeclaration|null;
```